Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertRailing.cs

```diff
@@ -67,6 +67,7 @@ public ApplicationObject RailingToNative(BuiltElements.Revit.RevitRailing speckl
 
         if (topRailType != null && isTopRailExactMatch)
           railingType.TopRailType = topRailType.Id;
```

Comment:
you know what I am going to {say} :)

Reply:
Agree, this was pre-existing code tho ;)

Reply:
All of this looked changed apart from the first one!